content/rules/efbfc7/proposed.md

```diff
@@ -90,8 +90,8 @@ There are no accessibility support issues known.
   <li><details>
     <summary><span>WCAG Non-Interference</span></summary>
     <ul>
-      <li><a href="https://www.w3.org/TR/WCAG21/#cc5">Learn more about WCAG Non-Interference</a></li>
-      <li><strong>Required for conformance</strong> to WCAG 2.1.</li>
+      <li><a href="https://www.w3.org/TR/WCAG22/#cc5">Learn more about WCAG Non-Interference</a></li>
+      <li><strong>Required for conformance</strong> to WCAG 2.2.</li>
```

Comment:
@daniel-montalvo Any reason not to use "Required for conformance to WCAG 2.0 and later"?